From ec77e7ae65094227e2afd7100287782aa8b0c88c Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Mon, 24 Dec 2018 01:03:47 +0200
Subject: [PATCH 21/28] extract_utils: drop the "/system/" prefix from prints
 of blob names

* This makes the printed output closer to the proprietary-files.txt syntax

Change-Id: I81b844bb6bb1d1a2f91a39151a892fbfc0bed20b
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 build/tools/extract_utils.sh | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index f5864a88..2639f761 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -1011,11 +1011,7 @@ function extract() {
             DST_FILE="/system/${SPEC_DST_FILE}"
         fi
 
-        if [ "$SRC" = "adb" ]; then
-            printf '  - %s .. \n' "${DST_FILE}"
-        else
-            printf '  - %s \n' "${DST_FILE}"
-        fi
+        printf '  - %s \n' "${DST_FILE#/system/}"
 
         # Strip the file path in the vendor repo of "system", if present
         local VENDOR_REPO_FILE="$OUTPUT_DIR/${DST_FILE#/system}"
-- 
2.17.1

